pkg/option/config.go

```diff
@@ -2121,7 +2123,7 @@ func (c *DaemonConfig) LocalClusterName() string {
 // service.kubernetes.io/service-proxy-name label in order for services to be
 // handled.
 func (c *DaemonConfig) K8sServiceProxyName() string {
-	return c.k8sServiceProxyName
+	return c.K8sServiceProxyNametest
```

Comment:
Mistake here.

Reply:
Yes sure, Also do we really need these public functions to return config value from the struct? The reason I am asking this is that  we are already making the configurations public.